File: android/app/src/main/java/com/nhs/online/nhsonline/data/PaycassoData.kt
package com.nhs.online.nhsonline.data

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class PaycassoData (
    val credentials: PaycassoCredentials,
    val externalReferences: PaycassoExternalReferences,
    val transactionDetails: PaycassoTransactionDetails
)

File: xamarin/NHSOnline.App.iOS/DependencyServices/PaycassoData.cs
namespace NHSOnline.App.iOS.DependencyServices
{
    public sealed class PaycassoData
    {
        public PaycassoCredentials Credentials { get; set; } = new PaycassoCredentials();
        public PaycassoExternalReferences ExternalReferences { get; set; } = new PaycassoExternalReferences();
        public PaycassoTransactionDetails TransactionDetails { get; set; } = new PaycassoTransactionDetails();
    }
}